experiments/run_bulk_evict_insert.py

```diff
@@ -15,12 +15,12 @@
 
 base_window_sizes = [4*u.MB]
 
-base_iterations = 10 * u.MILLION
+base_iterations = 5 * u.MILLION
 
 bulk_sizes = [
     1, 8, 64,
-    1*u.KB, 8*u.KB, 64*u.KB, 
-    1*u.MB,
+    #1*u.KB, 8*u.KB, 64*u.KB, 
```

Comment:
Are we now aiming to test less than before?

Reply:
No, again, this was just to get some sample data. First step before running any of these is going into the script and deciding what to run - which is why these scripts have so little logic, so they're more of a configuration file.